File: Python/1010.py
p1 = input()
p1_code, p1_units, p1_price = p1.split(' ')

p2 = input()
p2_code, p2_units, p2_price = p2.split(' ')

p1_units = int(p1_units)
p2_units = int(p2_units)

p1_price = float(p1_price)
p2_price = float(p2_price)

total = p1_units * p1_price + p2_units * p2_price

print("VALOR A PAGAR: R$ {:.2f}".format(total))

File: JavaScript/1010.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

var p1 = lines.shift();
p1 = p1.split(' ')

var p1_code = parseInt(p1[0])
var p1_units = parseInt(p1[1])
var p1_price = parseFloat(p1[2])

var p2 = lines.shift();
p2 = p2.split(' ')

var p2_code = parseInt(p2[0])
var p2_units = parseInt(p2[1])
var p2_price = parseFloat(p2[2])

total = p1_units * p1_price + p2_units * p2_price

console.log("VALOR A PAGAR: R$ " + total.toFixed(2))